esp32_azure_iot_kit/esp32_azure_iot_kit.c

```diff
@@ -30,6 +49,13 @@ typedef enum {
 
 esp_err_t bsp_i2c_init(void)
 {
+    static bool i2c_initialized = false;
```

Comment:
This variable should be static on this file level and set to `false` in bsp_i2c_deinit

Reply:
Done.